assets/js/index.js

```diff
@@ -0,0 +1,249 @@
+  $("#main")
+    .append(`      <section id="weatherContainer" class="col-sm-12 col-md-9 border">
+  <!-- current weather data -->
+  <div id="currentWeather" class="border row"></div>
+
+  <!-- forecast weather data -->
+    <!-- title -->
+   <h2 class="title">5-Day Forecast</h2>
+    <div
+      id="forecastWeather"
+      class="d-flex justify-content-between flex-wrap"
+    > 
+      <!-- forecast weather card 1 -->
+      <!-- <div class="card" style="width: 8rem">
+        <img src="..." class="card-img-top" alt="..." />
+        <div class="card-body">
+          <h5 class="card-title">Card title</h5>
+          <p class="card-text">
+            Some quick example text to build on the card title and make up
+            the bulk of the card's content.
+          </p>
+          <a href="#" class="btn btn-primary">Go somewhere</a>
+        </div>
+      </div> -->
+    </div>
+  </div>
+</section>`);
+  const target = $(event.target);
+
+  if (target.is("a")) {
+    console.log("click");
+    const search = $(target).attr("data-value");
+    renderWeatherData(search);
+  }
+};
+const handleFormSubmit = (event) => {
+  // get the city name from input
+  event.preventDefault();
+  //get text input
+  const container = document.getElementById("weatherContainer");
+  container.remove();
+  const main = document.getElementById("main");
+  $("#main")
+    .append(`      <section id="weatherContainer" class="col-sm-12 col-md-9 border">
+  <!-- current weather data -->
+  <div id="currentWeather" class="border row"></div>
+
+  <!-- forecast weather data -->
+    <!-- title -->
+   <h2>5-Day Forecast</h2>
+    <div
+      id="forecastWeather"
+      class="d-flex justify-content-between flex-wrap"
+    > 
+      <!-- forecast weather card 1 -->
+      <!-- <div class="card" style="width: 8rem">
+        <img src="..." class="card-img-top" alt="..." />
+        <div class="card-body">
+          <h5 class="card-title">Card title</h5>
+          <p class="card-text">
+            Some quick example text to build on the card title and make up
+            the bulk of the card's content.
+          </p>
+          <a href="#" class="btn btn-primary">Go somewhere</a>
+        </div>
+      </div> -->
+    </div>
+  </div>
+</section>`);
+  let search = document.getElementById("input-text").value;
+  storeInLS("recentSearches", search);
+  if (search) {
+    //validate
+
+    // else render weather data
+    renderWeatherData(search);
+    renderRecentCities();
+    // add search to recent searches local storage
+    // if city name is empty handle that
+  } else {
+    alert("enter valid search");
+  }
+};
+// function to store answer in local storage
+const storeInLS = (key, value) => {
+  //get feedbackResults from LS
+  const arrayFromLS = JSON.parse(localStorage.getItem(key));
+
+  //push answer in to array
+  arrayFromLS.push(value);
+
+  //set feedbackResults in LS
+  localStorage.setItem(key, JSON.stringify(arrayFromLS));
+};
```

Comment:
Please make sure you leave spare lines between your functions to make your code more readable.